### PR TITLE
Add OSVDB-126329, OSVDB-126330, and OSVDB-126331 for sidekiq-pro

### PR DESCRIPTION
**Pull request by reedloden**
https://github.com/mperham/sidekiq/blob/master/Pro-Changes.md

These are for the closed-source `sidekiq-pro` gem (https://github.com/mperham/sidekiq-pro/).

### CODE REVIEW

File: gems/sidekiq-pro/OSVDB-126331.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+gem: sidekiq-pro
+osvdb: 126331
+url: http://osvdb.org/show/osvdb/126331
+title: Sidekiq Pro Gem for Ruby CSRF in Job Filtering
+date: 2015-07-17
+description: |
+  Sidekiq::Web job filtering lacks CSRF protection. This issue
+  is related to OSVDB-125675.
```
Comment (reedloden) [inline gems/sidekiq-pro/OSVDB-126331.yml]: Do we want to use the (undocumented) `related` field for this? Basically, Sidekiq Pro depends on Sidekiq, so the fix in Sidekiq (OSVDB-125675) is needed for this fix in Sidekiq Pro to work correctly.

Comment (phillmv) [inline gems/sidekiq-pro/OSVDB-126331.yml]: After investigating this, sounds about right +1

**Comment by phillmv** [comment]
Any reason the url points to osvdb? Right now those pages are unallocated but https://github.com/mperham/sidekiq/issues/2442 seems to be a _little_ meaningful. Basically, is there any better context?

**Comment by reedloden** [comment]
I can link to the actual commits in the private sidekiq-pro repo, but those aren't publicly available. Could also link to the public changelog updates...

https://github.com/mperham/sidekiq/commit/651400ed8f237118346895c99dc28ca94f3169d3
https://github.com/mperham/sidekiq/commit/a695ff347ae50f641dfc35189131b232ea0aa1db
https://github.com/mperham/sidekiq/commit/99b12fb50fe244c5a317f03f1bed9b333ec56ebe

Thoughts?

File: gems/sidekiq-pro/OSVDB-126331.yml (hunk 1 above)
Comment (reedloden) [inline gems/sidekiq-pro/OSVDB-126331.yml]: Added.

**Comment by reedloden** [comment]
Poke on the above, @phillmv

**Comment by phillmv** [comment]
Was there no announcement or blog post?

Between the changelog and a non functional osvdb link I'll take the changelog.

Thanks, Reed!

**Comment by reedloden** [comment]
@phillmv nothing public, afaik. Maybe something privately e-mailed to Sidekiq Pro subscribers (predates my use of the gem). Updated to use the changelog commits.

**Pull request merged**